deploying_changes/production.md

```diff
@@ -2,15 +2,17 @@
 
 #### Tools Used
 
- - `ep merge`
+ - GitHub
  - `ep deploy`
+ - `ep release`
 
 
 #### Procedure
 
  - Project Maintainers regularly merge pull requests that have passed testing (or don't require testing) and have passed code review, and deploy them to Production.
-     - Branches are merged using `ep merge <branch-name>`
+     - Branches are merged using the "Merge" button on the Pull Request in GitHub
      - The `master` branch is deployed using `ep deploy production`
+     - The deploy is recorded using `ep release` to generate a release with release notes in GitHub
  - In general, deploys to Production should take place frequently; but there are two kinds of pull requests that cannot be deployed immediately:
      1. PRs that not ["Hot Compatible"](developing_features/hot_compatibility.md) (i.e. that require a maintenance page)
```

Comment:
This is outside the scope of your changes, but should this read:
> PRs that _are_ not "Hot Compatible"

Reply:
Good catch. I went ahead and amended the commit with that change. 👍